common/models/access-token.js

```diff
@@ -171,7 +171,8 @@ module.exports = function(AccessToken) {
     cookies = cookies.concat(['access_token', 'authorization']);
 
     for (length = params.length; i < length; i++) {
-      id = req.param(params[i]);
+      // Only look up params in query. There is no body here.
+      id = req.query && req.query[params[i]];
```

Comment:
I'm refactoring this, but there is one test-case that mocks a request that does _not_ contain `req.params` or `req.query`. I've contacted @bajtos since it's his test.